src/devices/Mcp3428/Mcp3426.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Device.I2c;
+
+namespace Iot.Device.Mcp3428
+{
+    public class Mcp3426 : Mcp3428
+    {
+        public const int Address = 0x68;
+
+        /// <inheritdoc />
+        public Mcp3426(I2cDevice i2CDevice) : base(i2CDevice, 2)
```

Comment:
please create private constant for 2 since it's not immediately obvious (without looking at the base)  what that is